scripts/core/services/pageTitle.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * PageTitle service
+ *
+ * PageTitle service is used to save and update the title of the page.
+ * The title will be 'Superdesk {- Url} {-Desk/Workspace}
+ * Url can be: Dashboard, Monitoring, Spike, Highlights, Search, Settings
+ * If Url is search then there won't be desk
+ *
+ */
+export default angular.module('superdesk.core.services.pageTitle', [])
+    .service('pageTitle', function() {
+        this.title = 'Superdesk';
+        this.url = '';
+        this.workspace = '';
+
+        this.setUrl = function(url) {
```

Comment:
I agree with you that setUrl sounds more like a navigation action if you see the method name in isolation. But since the context of the function is pageTitle service then it won't be much of an issue. Nonetheless, I'm changing the name to something less confusing.